3-0-spring-mvc/3-0-1-hello-spring-mvc/src/test/java/ControllerTest.java

```diff
@@ -0,0 +1,88 @@
+import com.bobocode.mvc.HelloSpringMvcApp;
+import com.bobocode.mvc.model.Note;
+import com.bobocode.mvc.storage.Notes;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest(classes = HelloSpringMvcApp.class)
+@AutoConfigureMockMvc
+public class ControllerTest {
+
+    @Autowired
+    private Notes notes;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void returnNoteListModelWhenGetRequestPerformed() throws Exception {
```

Comment:
@erneman1 please use testg naming convention when test method name start from method name that is under tests...
e.d. `getAllNotesReturnsNoteList` or in this case test can be called just `getAllNotes`